subsys/bluetooth/mesh/cfg_srv.c

```diff
@@ -2305,7 +2305,7 @@ static int heartbeat_pub_set(struct bt_mesh_model *model,
 
 	pub.dst = sys_le16_to_cpu(param->dst);
 	if (param->count_log == 0x11) {
-		/* Special case defined in Mesh Profile Errata 11737 */
+		/* Special case defined in MshPRT-v1.1 Errata 11737 */
```

Comment:
I'm uncertain about this. I guess the erratum was created to 1.0.1 spec, not to 1.1.

Reply:
I found the same errata number (11737) in 1.1 spec as well. My thought was that it is easier to reference as little to the 1.0.1 as possible so that the reader mainly can reference the new spec.

Reply:
It is mentioned in the MshPRTv1.1 but as I understand it was filed to 1.0.1. But now I read this as it was filed to 1.1 spec.